pkg/cmd/propertiesSet.go

```diff
@@ -42,7 +42,8 @@ func init() {
 	propertiesSetCmd.MarkPersistentFlagRequired("name")
```

Comment:
Not in your PR but since we're adding the "name" flag below, I think we can remove this `MarkPersistentFlagRequired` line